scripts/travis_test.sh

```diff
@@ -108,9 +129,11 @@ run_examples() {
 case $1 in
     ethereum_vm)
         make_vmtests
+        install_truffle
+        RV=$?
         echo "Running only the tests from 'tests/$1' directory"
         run_tests_from_dir $1
```

Comment:
@montyly Are you missing a call to `run_truffle_tests`?

Reply:
Indeed, my bad